examples/gallery/symbols/custom_symbols.py

```diff
@@ -0,0 +1,49 @@
+"""
+Custom symbols
+--------------
+
+The :meth:`pygmt.Figure.plot` method can plot individual custom symbols
+by passing the corresponding symbol name together with the **k** shortcut to
+the ``style`` parameter. In total 41 custom symbols are already included of
+which the following plot shows five examplary ones. The symbols are shown
+underneath their corresponding names. For the remaining symbols see the GMT
+cookbook :gmt-docs:`cookbook/custom-symbols.html`.
+"""
+
+import pygmt
+
+fig = pygmt.Figure()
+fig.basemap(region=[0, 8, 0, 3], projection="X12c/4c", frame=True)
+
+# define pen and fontstlye for annotations
+pen = "1p,black"
+font = "15p,Helvetica-Bold"
+
+y = 1.25
+
+# use the volcano symbol with a size of 60p,
+# fill color is set to "seagreen"
+fig.plot(x=1, y=y, style="kvolcano/60p", pen=pen, color="seagreen")
+fig.text(x=1, y=y + 1.25, text="volcano", font=font)
```

Comment:
Looks good. 

Two suggestions:

1. Can you slightly decrease the symbol size to make it less crowded.
2. Although `60p` is a valid symbol size, usually it's recommended to use **c**entimeter for symbol size, and use **p**oint for pen size. Better to change it to a size like `2.0c`.

Reply:
Done! Thanks @seisman.